Kernel/RobSubstitution.hpp

```diff
@@ -46,7 +46,7 @@ class RobSubstitution
   CLASS_NAME(RobSubstitution);
   USE_ALLOCATOR(RobSubstitution);
   
-  RobSubstitution() : _nextUnboundAvailable(0) {} //,_nextAuxAvailable(0) {}
+  RobSubstitution() : _funcSubtermMap(nullptr), _nextUnboundAvailable(0) {}
```

Comment:
POD types are not automatically initialised in constructors. This will cause a crash more readily if `_funcSubtermMap` is not set before use somehow.